src/components/lab/canvas/Cloud.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Group, Circle, Ellipse } from 'react-konva';
+import {
+  CLOUD_BORDER_COLOR,
+  CLOUD_BORDER_WIDTH,
+  CLOUD_CENTRAL_CIRCLE_RADIUS,
+  CLOUD_CENTRAL_CIRCLE_X,
+  CLOUD_CENTRAL_CIRCLE_Y,
+  CLOUD_ELLIPSE_RADIUS_X,
+  CLOUD_ELLIPSE_RADIUS_Y,
+  CLOUD_FILL,
+  CLOUD_RESPONSIVE_ADJUSTMENT_FACTOR,
+} from '../../../config/constants';
+import { generateCloudCircles } from '../../../utils/canvas';
+
+const Cloud = ({ skyHeight, skyWidth, skyBeginsY }) => {
+  const centralCircleRadius =
+    CLOUD_CENTRAL_CIRCLE_RADIUS *
+    ((skyHeight + skyWidth) / CLOUD_RESPONSIVE_ADJUSTMENT_FACTOR);
```

Comment:
What's the logic behind it? Can't you use only a percent of the sky height?

Reply:
Yes this is a bit weird... I basically did it so that the cloud is responsive when you adjust screen size both horizontally and vertically. If we use just `skyHeight`, then it will only adjust to vertical changes (similar to how the sun is currently). At some point let's look at/assess how nice the canvas elements' responsive-ness is.

I added a comment in the code to clarify.